LibCarla/source/carla/trafficmanager/LocalizationUtils.cpp

```diff
@@ -5,16 +5,19 @@
 // For a copy, see <https://opensource.org/licenses/MIT>.
 
 #include "carla/trafficmanager/LocalizationUtils.h"
+#include "carla/trafficmanager/Constants.h"
+
 
 namespace carla {
 namespace traffic_manager {
 
+using constants::Collision::EPSILON;
+
 float DeviationCrossProduct(const cg::Location &reference_location,
                             const cg::Vector3D &heading_vector,
                             const cg::Location &target_location) {
   cg::Vector3D next_vector = target_location - reference_location;
-  float vector_magnitude_epsilon = 2.0f * std::numeric_limits<float>::epsilon();
-  next_vector = next_vector.MakeSafeUnitVector(vector_magnitude_epsilon);
+  next_vector = next_vector.MakeSafeUnitVector(EPSILON);
```

Comment:
As you  are passing always `EPSILON` maybe make it a default on `MakeSafeUnitVector` ?